packages/design-system/src/styles/components/_Button.scss

```diff
@@ -182,22 +182,25 @@
 %transparent-inverse-button {
   border-color: transparent;
   background-color: transparent;
+
   // Default inverse focus styles are overridden above
   @if $ds-include-focus-styles {
     &:focus,
     &.ds-c-button--focus {
       border-color: transparent;
+      background-color: $focus-color;
+      color: $color-base;
     }
   }
 
-  &:disabled,
-  &:hover,
-  &:active,
-  &.ds-c-button--disabled,
-  &.ds-c-button--hover,
-  &.ds-c-button--active {
-    border-color: transparent;
```

Comment:
Why was this deleted? And it looks like there's a leftover comment here

Reply:
This was removed because the transparent border is already declared up on line 183 for the transparent inverse buttons so it was not needed here.